torch/ao/quantization/pt2e/export_utils.py

```diff
@@ -195,7 +195,12 @@ def _move_exported_model_to_eval(model: torch.fx.GraphModule):
 
     This is equivalent to model.eval() but only for certain special ops like dropout, batchnorm.
     QAT users should call this before performing inference on the model.
+
+    This call is idempotent; if the model is already in eval mode, nothing will happen.
     """
+    if not model.training:
+        return model
+    model.training = False
```

Comment:
is this attribute preserved in the transformations after export, will it be a problem if it's not?

Reply:
Hmm good point, maybe safer to use a different private flag